autogpt_platform/backend/backend/notifications/email.py

```diff
@@ -62,7 +62,7 @@ def send_templated(
                 subject_template=template.subject_template,
                 content_template=template.body_template,
                 data=data,
-                unsubscribe_link="https://autogpt.com/unsubscribe",
+                unsubscribe_link="https://platform.agpt.co/profile/settings",
```

Comment:
Shouldn't this be an un authenticated link with some token in it?
Otherwise the user will need to log in and will likely block us instead or report it as spam.

Reply:
Based on chat with john, thats a future goal